music/src/core/player.ts

```diff
@@ -47,9 +47,14 @@ function compareQuantizedNotes(a: NoteSequence.INote, b: NoteSequence.INote) {
  * played.
  */
 export abstract class BasePlayerCallback {
-  /* Will be called for each time/note pair in a sequence being played.
+  /**
+   *   Will be called for each time/note pair in a sequence being played.
+   *
+   *   @param n A NoteSequence.INote object, which is the note being played
+   *     at the moment.
+   *   @param t A number, the time at which the note is being played.
    */
```

Comment:
No need to specify the type since it is specified below and added to generated documentation automatically.

Reply:
done.